CMakeLists.txt

```diff
@@ -385,7 +385,7 @@ cmake_dependent_option(IREE_TARGET_BACKEND_CUDA "Enables the 'cuda' compiler tar
 # Non-default target backends either have additional dependencies or are
 # experimental/niche in some fashion.
 # Disable WebGPU by default - it has complex deps and is under development.
-cmake_dependent_option(IREE_TARGET_BACKEND_WEBGPU "Enables the 'webgpu' compiler target backend" OFF ${IREE_BUILD_COMPILER} OFF)
+cmake_dependent_option(IREE_TARGET_BACKEND_WEBGPU_SPIRV "Enables the 'webgpu' compiler target backend" OFF ${IREE_BUILD_COMPILER} OFF)
```

Comment:
We could forward the old option to the new option if we wanted to ease the transition. Probably fine to just push forward though. I know I'll have a few local configs to update, but I don't expect others to really be impacted.

Here's some (verbose) forwarding we used before: https://github.com/openxla/iree/blob/adeb538647f1b8b588fd4d1d115df4f8ed3e0fd7/CMakeLists.txt#L86-L94

Reply:
good idea - if literally anyone were using this I'd do it, but you may be the only human to compile this today :)